src/pages/forms/UpdateProfile.js

```diff
@@ -0,0 +1,223 @@
+                  <Input
+                    labelName="Address"
+                    defaultValue={addressInfo}
+                    type="text"
+                    name={address}
+                    {...register('address', {
+                      required: 'address is required',
+                      pattern: /^(?!.* {2})[a-z][a-z ]{3,}/i
+                    })}
+                  />
+                  <div className="h-8">
+                    {errors.address && (
+                      <p className="mb-4 error-message required">
+                        Invalid address
+                      </p>
+                    )}
+                  </div>
+                </div>
+              </div>
+              <div className="lg:ml-4 xl:ml-4 lg:w-96 xl:w-96">
+                <div>
+                  <Input
+                    labelName="Last name"
+                    defaultValue={lastNameInfo}
+                    type="text"
+                    name={lastName}
+                    {...register('lastName', {
+                      required: 'Last name is required',
+                      pattern: /^[a-zA-Z][a-zA-Z ]{3,32}$/i
+                    })}
+                  />
+                  <div className="h-8">
+                    {errors.lastName && (
+                      <p className="mb-4 error-message required">
+                        A valid last name is required
+                      </p>
+                    )}
+                  </div>
+                </div>
+                <div>
+                  <div className="mb-2 block font-raleway">
+                    <label
+                      htmlFor="Company"
+                      className="block text-grey-darker text-sm font-bold font-rale  mb-2"
+                    >
+                      Company
+                    </label>
+                    <select
+                      id="company"
+                      name={company}
+                      defaultValue={companyInfo}
+                      {...register('company', {
+                        required: true
+                      })}
+                      className="border font-rale rounded w-full py-2 px-3 text-grey-darker bg-gray-200 text-md outline-hidden mb-1"
+                    >
+                      <option hidden>Select Company</option>
+                      <option value="REL">Royal Express Limited</option>
+                      <option value="CEL">City Express Limited</option>
+                      <option value="KBS">Kigali Bus Services</option>
+                    </select>
+                  </div>
+                  <div className="h-8">
+                    {errors.company && (
+                      <p className="error-message required">
+                        Company name is required
+                      </p>
+                    )}
+                  </div>
+                </div>
+                <div>
+                  <Input
+                    labelName="National ID"
```

Comment:
Since **National ID** is a unique identifier, It should not be edited by the driver or its owner. 

Even in case of loss of an ID card. The previous number remain active and linked to the individual which make me believe that editing ID should also be disabled. 
Instead the ability to change the **phone** would be great as number change so often.

Reply:
You are right. Will change this